### PR TITLE
Update ome.md

### PR DESCRIPTION
**Pull request by msbemba**
tested and removed outlook.com as the experience is seamless

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7748

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/ome.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ For a detailed list of the differences between the previous version of OME and t
 
 When someone sends an email message that matches an encryption mail flow rule, the message is encrypted before it's sent. All Microsoft 365 end users that use Outlook clients to read mail receive native, first-class reading experiences for encrypted and rights-protected mail even if they're not in the same organization as the sender. Supported Outlook clients include Outlook desktop, Outlook Mac, Outlook mobile on iOS and Android, and Outlook on the web (formerly known as Outlook Web App).
 
-Recipients of encrypted messages who receive encrypted or rights-protected mail sent to their Outlook.com, Gmail, and Yahoo accounts receive a wrapper mail that directs them to the OME Portal where they can easily authenticate using a Microsoft account, Gmail, or Yahoo credentials.
+Recipients of encrypted messages who receive encrypted or rights-protected mail sent to their Gmail, and Yahoo accounts receive a wrapper mail that directs them to the OME Portal where they can easily authenticate using a Microsoft account, Gmail, or Yahoo credentials.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ome.md]:
```suggestion
Recipients of encrypted messages who receive encrypted or rights-protected mail sent to their Gmail and Yahoo accounts receive a wrapper mail that directs them to the OME Portal where they can easily authenticate using a Microsoft account, Gmail, or Yahoo credentials.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@KCCross Gentle reminder

**Comment by yogkumgit** [comment]
@KCCross Kindly merge this PR. Thanks

**Comment by KCCross** [comment]
@samschan-ms - Review the situations where Microsoft accounts end up with an OPC. @msbemba - Requires more context.

**Comment by samschan-ms** [comment]
@KCCross this is correct. Outlook.com can render encrypted mail inline.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by yogkumgit** [comment]
@msbemba Please add a comment.

**Comment by msbemba** [comment]
@yogkumgit Sam has confirmed that Outlook.com can render encrypted mail inline. It is seamless per my testing as well.

We can check with KC Cross to see if this can be merged now.

**Comment by yogkumgit** [comment]
@KCCross Please comment or merge this PR. Thanks

**Pull request merged**